data/json/items/comestibles/carnivore.json

```diff
@@ -315,10 +281,7 @@
     "spoils_in": 0,
     "quench": -1,
     "fun": 5,
-    "proportional": {
-      "price": 20,
-      "calories": 0.95
-    },
+    "proportional": { "price": 20, "calories": 0.95 },
```

Comment:
I guess its a bit more readable if ywe would keep all these entries multiline.

Reply:
I agree but I'm not sure how to go about changing the linter.

Reply:
Yeah, thats a thing for a different PR. I will try to look into it.